File: app/src/main/java/com/jcaseydev/bart/Model2/Fare/FareCost.java
package com.jcaseydev.bart.Model2.Fare;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class FareCost {

  @SerializedName("root")
  @Expose
  private Root root;

  public Root getRoot() {
    return root;
  }

  public void setRoot() {
    this.root = root;
  }

  public String getOrigin() {
    return getRoot().getOrigin();
  }

  public String getDestination() {
    return getRoot().getDestination();
  }

  public String getFare() {
    return getRoot().getTrip().getFare();
  }

}

File: app/src/main/java/com/jcaseydev/bart/Model2/Fare/FareCost.kt
package com.jcaseydev.bart.Model2.Fare

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class FareCost {

    @SerializedName("root")
    @Expose
    var root: Root? = null
        private set

    val origin: String
        get() = root!!.origin

    val destination: String
        get() = root!!.destination

    val fare: String
        get() = root!!.trip.fare

    fun setRoot() {
        this.root = root
    }

}
